README.md

```diff
@@ -46,14 +46,56 @@ kernel while others are delegated to the target.
 
 # Installation Guide
 
-## From Binaries
+## Verify the Intel SGX DCAP Driver is Installed
 
-You can download the latest binary release [from our builds
-page](https://github.com/deislabs/mystikos/releases)
+Some distributions come with the SGX driver already installed; if it is,
+you don't need to re-install it. You can verify this by running:
 
-**TODO**: Include installation instructions. 
+```bash
+dmesg | grep -i sgx
+```
 
-## From Source
+If the output is blank, install the driver manually by downloading it from Intel.
+
+> NOTE: The script below may not refer to the latest Intel SGX DCAP driver.
+> Check [Intel's SGX Downloads page](https://01.org/intel-software-guard-extensions/downloads)
+> to see if a more recent SGX DCAP driver exists.
+
+```bash
+sudo apt -y install dkms
+wget https://download.01.org/intel-sgx/sgx-dcap/1.7/linux/distro/ubuntu18.04-server/sgx_linux_x64_driver_1.35.bin -O sgx_linux_x64_driver.bin
+chmod +x sgx_linux_x64_driver.bin
+sudo ./sgx_linux_x64_driver.bin
+```
+## Add Intel's repository & install the sgx libraries
+
+```bash
+echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list
+wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -
+
+sudo apt update
+
+sudo apt -y install libsgx-enclave-common libsgx-dcap-ql libsgx-dcap-ql-dev
```

Comment:
Might be worth pointing out that for ACC VMs, az-dcap-client is a required package for remoted attestation.

Reply:
Wouldn't that be better in an Azure-specific example, perhaps in the docs directory?

Other parts of this README are not cloud specific (and even call out that this can be tested on _any_ sgx-capable hardware).

Reply:
Without az-dcap-client, a bunch of tests/solutions would fail. For this reason, we prefer users of Mystikos to get an ACC VM. We can put the note here or in another document. I am fine either way.